internal/backends/winit/renderer/femtovg/itemrenderer.rs

```diff
@@ -112,9 +112,8 @@ fn adjust_rect_and_border_for_inner_drawing(
     *border_width = border_width.min(rect.width_length() / 2.);
     // adjust the size so that the border is drawn within the geometry
 
-    let half_border_size = PhysicalSize::from_lengths(*border_width / 2., *border_width / 2.);
-    rect.origin += half_border_size;
-    rect.size -= half_border_size;
+    rect.origin += PhysicalSize::from_lengths(*border_width / 2., *border_width / 2.);
```

Comment:
Why `/2.` ? 
Is it because femtovg and skia are drawing the border like that?
I'd argue that it shouldn't be like this and the border should be drawn within the geometry. But if both femtovg and skia do the same, then that's just how we have to do.
But in the future as we want to draw the background on the full rectangle, we will have to do something about that.

Reply:
As discussed in the call, this change brings FemtoVG and Skia in line with Qt (restoring behavior). I'm going to file a new ticket to discuss the interaction of the border with the background and the clipping.

Reply:
Filed https://github.com/slint-ui/slint/issues/1988